docs/sites.yml

```diff
@@ -6405,3 +6405,16 @@
   built_by: Kevin Hirczy
   built_by_url: https://nehalist.io
   featured: false
+- title: March And Ash
+  main_url: https://marchandash.com/
+  url: https://marchandash.com/
+  description: >-
+    March and Ash is a customer-focused, licensed cannabis dispensary located in Mission Valley.
+  categories:
+    - eCommerce
+    - Business
+    - CBD
+    - Blog
```

Comment:
I couldn't find the blog on your website. Mind removing this tag or point me to the blog ^^?

Reply:
Hey @wardpeet, you can find the majority of the blog posts here https://marchandash.com/discover